lib/usage-tracking/class-usage-tracking-base.php

```diff
@@ -308,6 +309,70 @@ public function add_usage_tracking_two_week_schedule( $schedules ) {
 		return $schedules;
 	}
 
+	/**
+	 * Collect system data to track.
+	 *
+	 * @return array
+	 */
+	public function get_system_data() {
+		global $wp_version;
+
+		/**
+		 * @var WP_Theme $theme Current active theme.
+		 */
+		$theme = wp_get_theme();
+
+		$system_data                         = array();
+		$system_data['wp_version']           = $wp_version;
+		$system_data['php_version']          = PHP_VERSION;
+		$system_data['locale']               = get_locale();
+		$system_data['multisite']            = is_multisite() ? 1 : 0;
+		$system_data['active_theme']         = $theme['Name'];
+		$system_data['active_theme_version'] = $theme['Version'];
+
+		$plugin_data = $this->get_plugin_data();
+		foreach ( $plugin_data as $plugin_name => $plugin_version ) {
```

Comment:
@donnapep Do we have a limit on how big our query string can be? If so this could be an issue for sites that have many plugins.

Reply:
Yeah, there's definitely a limit there. I wonder if perhaps we just don't worry about the version of other plugins for now? We can solve that query limit problem later on.